test/specs/builder/src/helpx_html.pre.js

```diff
@@ -28,8 +28,14 @@ function collectMetadata(req, logger) {
 
   logger.debug(`Fetching Git Metadata from ${options.uri}`);
 
-  return request(options)
-    .then((metadata) => {
+  return fetch(options.uri, options)
+    .then(async (result) => {
```

Comment:
nitpicking... `fetch` returns a `Response` object. `resp` instead of `result` would make it more obvious.